src/Application/Application.csproj

```diff
@@ -1,12 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>net8.0</TargetFramework>
+		<TargetFramework>net9.0</TargetFramework>
```

Comment:
Move common properties to `Build.Directory.props` so that you only need to update `TargetFramework` in one place next time